### PR TITLE
add compute.toys

### PR DESCRIPTION
**Pull request by munrocket**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -131,7 +132,6 @@ Demos might work only on Chrome. Firefox implementation is not complete.
 - [WebGPU Metaballs](https://toji.github.io/webgpu-metaballs/) - by [Brandon Jones](https://github.com/toji) - [repository](https://github.com/toji/webgpu-metaballs)
 - [WebGPU External Texture Test](https://toji.github.io/webgpu-external-test/) - by [Brandon Jones](https://github.com/toji) - [repository](https://github.com/toji/webgpu-external-test)
 - [Online WGSK Editor](https://takahirox.github.io/online-wgsl-editor/) - by [Takahiro](https://github.com/takahirox) - [repository](https://github.com/takahirox/online-wgsl-editor)
-- [Safari WebKit demos](https://webkit.org/demos/webgpu/)
```
Comment (mikbry) [inline readme.md:134]: @munrocket why did delete this line ?

File: readme.md [hunk 2]
```diff
@@ -32,6 +32,7 @@
 - [API quick reference and documentation](https://webgpu.rocks/) - WebGPU.rocks
 - [107 WebGPU Projects on GitHub](https://awesomeopensource.com/projects/webgpu) - AwesomeOpenSource.com
 - [r/WebGPU - Reddit](https://www.reddit.com/r/webgpu/) - WebGPU Subreddit
+- [webgpu.toys](https://webgpu.toys/) - Compute shader playground (like shadertoy)
```
Comment (mikbry) [inline readme.md:35]: @munrocket the link is not working, it should be https://compute.toys/ ?

**Review by mikbry — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (munrocket) [reply readme.md:134]: They removed support of this yearly days demos in Safari TP. It is literally impossible to check anywhere.

File: readme.md (hunk 2 above)
Comment (munrocket) [reply readme.md]: Sorry my bad, you right.

**Review by mikbry — APPROVED** [review]

**Comment by mikbry** [comment]
Thanks @munrocket 👍 !

**Pull request merged**